website/docs/search.md

```diff
@@ -37,15 +37,15 @@ By default, the Docusaurus preset generates a [sitemap.xml](https://docusaurus.i
 
 :::
 
-:::info From an old docsearch version?
+:::info From the old docsearch?
 
-You can read more about migration from DocSearch v2 and configurating v3 in [our blog post](/blog/2021/11/21/algolia-docsearch-migration).
+You can read more about migration from the legacy DocSearch infra in [our blog post](/blog/2021/11/21/algolia-docsearch-migration).
```

Comment:
I know it's already linked in the blog post, but we also have this migration guide here: https://docsearch.algolia.com/docs/migrating-from-legacy